cob_bringup_sim/scripts/spawn_object.py

```diff
@@ -61,19 +61,26 @@
 import copy
 import math
 
-from gazebo_msgs.srv import SpawnModel, SpawnModelRequest, DeleteModel, DeleteModelRequest
+from spawn_model import SpawnModel
 from geometry_msgs.msg import Pose
 import tf.transformations as tft
 
 parents = {}
-compound_keys={}
+compound_keys = {}
+visualizedRooms = {}
```

Comment:
spawn_objects didn't know about "rooms" so far. This seems to be an either project specific or environment specific extension, so I like to NOT merge this.